src/main/java/com/teamknp/hotel/entity/KeyStatus.java

```diff
@@ -0,0 +1,18 @@
+package com.teamknp.hotel.entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+@Entity
+@Data
+public class KeyStatus {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    int id;
+    LocalDate timeGiven;
+    LocalDate timeReturned;
+    @OneToMany
+    @JoinTable
+    ReservationRoom reservationRoom;
```

Comment:
`OneToMany` zmień na `ManyToOne`. Zapamiętaj, że patrzy się na drugie słowo w relacji `OneTo>>Many<<` i jeśli ma się Many to jest to Lista/Set, a w przypadku One to jest to zwykła klasa.